engine/src/main/java/org/terasology/rendering/world/WorldRendererImpl.java

```diff
@@ -570,7 +569,7 @@ private boolean renderLightComponent(LightComponent lightComponent, Vector3f lig
             program.setFloat3("lightColorAmbient", lightComponent.lightColorAmbient.x, lightComponent.lightColorAmbient.y, lightComponent.lightColorAmbient.z, true);
 
             program.setFloat4("lightProperties", lightComponent.lightAmbientIntensity, lightComponent.lightDiffuseIntensity,
-                    lightComponent.lightSpecularIntensity, lightComponent.lightSpecularPower, true);
+                    0.0f, lightComponent.lightSpecularPower, true);
```

Comment:
Good point.  This was just to remove the value.  Thinking about this now,  I should make it a vec3 in the shader and then just avoid passing a value alltogether. (tracked in the PR checklist)